vyper/venom/basicblock.py

```diff
@@ -381,7 +381,7 @@ def __repr__(self) -> str:
         opcode = f"{self.opcode} " if self.opcode != "store" else ""
         s += opcode
         operands = self.operands
-        if opcode not in ["jmp", "jnz", "invoke"]:
+        if opcode not in ("jmp", "jnz", "invoke"):
```

Comment:
I now this is not necessarily part of the PR but there it should be `self.opcode` no? since the opcode contains space after original opcode so body of this `if` is unreachable right now (I have noticed it when thinking about order of operands in jnz)

Reply:
yea seems so. let's leave it out of scope, can be fixed in https://github.com/vyperlang/vyper/pull/4402 or https://github.com/vyperlang/vyper/pull/4403